src/test/java/org/example/utils/BaseTest.java

```diff
@@ -30,4 +31,8 @@ public CartPageBase getCartPage() {
     public CheckoutPageBase getCheckoutPage() {
         return initPage(getDriver(), CheckoutPageBase.class);
     }
+
+    public ProductPageBase loginToAccount() {
```

Comment:
loginAsStandardUser

Reply:
public ProductListPageBase loginAsStandardUser(boolean useAutoFilling) {
        LoginPageBase loginPageBase = initPage(LoginPageBase.class);
        ProductListPageBase productsPageBase = loginPageBase.login(UserType.STANDARD, useAutoFilling);
        Assert.assertTrue(productsPageBase.isPageOpened(), "Product page is not opened");
        return productsPageBase;
    }